From dd5d54a495d3a59e72307f07feeeb2ed1e2e24ce Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sat, 20 Oct 2018 17:59:29 -0600
Subject: [PATCH 4/8] klte-common: sepolicy: Resolve fingerprint HAL denials

* avc: denied { write } for name=fpdata dev=mmcblk0p26 ino=106076
  scontext=u:r:hal_fingerprint_default:s0
  tcontext=u:object_r:fingerprintd_data_file:s0 tclass=dir
  permissive=1

Change-Id: I624acde27d157daa473179980af30abd82b51131
---
 sepolicy/common/hal_fingerprint_default.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sepolicy/common/hal_fingerprint_default.te b/sepolicy/common/hal_fingerprint_default.te
index 8c06be7..0b99c8a 100644
--- a/sepolicy/common/hal_fingerprint_default.te
+++ b/sepolicy/common/hal_fingerprint_default.te
@@ -1,5 +1,6 @@
 r_dir_file(hal_fingerprint_default, firmware_file)
 
+allow hal_fingerprint_default fingerprintd_data_file:dir w_dir_perms;
 allow hal_fingerprint_default tee_device:chr_file rw_file_perms;
 allow hal_fingerprint_default vfsspi_data_file:dir rw_dir_perms;
 allow hal_fingerprint_default vfsspi_data_file:file create_file_perms;
-- 
2.17.1

